crates/polars-core/src/chunked_array/logical/categorical/builder.rs

```diff
@@ -605,27 +622,29 @@ mod test {
             Some("bar"),
         ];
         let ca = Utf8Chunked::new("a", slice);
-        let out = ca.cast(&DataType::Categorical(None))?;
+        let out = ca.cast(&DataType::Categorical(None, CategoricalOrdering::Physical))?;
```

Comment:
Then this all can be replaced with `Default::default()`. If in the future we choose to replace defaults this saves us a lot of work.